tests/autoyast/installation.pm

```diff
@@ -16,6 +16,7 @@
 use strict;
 use base 'basetest';
 use testapi;
+use utils qw(clear_console);
 
```

Comment:
I would be happy to change it after we noted down the development guidelines regarding this somewhere. Should I also change all other uses of `use <package> qw(…);` then?